### ISSUE TITLE
couldn''t find procedure Itcl_Init

### ISSUE THREAD

**Issue by pawelsalawa** [pawelsalawa/sqlitestudio#1480]
_(This issue was migrated from the old bug tracker of SQLiteStudio)_

Original ID from old bug tracker: 1502
Originally created at: Sat Jan  5 00:50:27 2013
Originally last updated at: Sat Jan  5 00:50:27 2013

I am running on a variant of Ubuntu 10.04 (Backtrack Linux) as the root user (shouldn''t matter, maybe it does)
I am getting the following error when trying to run the 2.1.0 binary (and actually the same error when trying 2.0.28 - 2.0.22). I have tried the same binary on Ubuntu 12.04 and it works just fine. Any idea why it won''t load on Backtrack?

couldn''t find procedure Itcl\\_Init
    while executing
&quot;load /sqlitestudio-2.1.0.bin/lib/itcl3.4/libitcl3.4.so Itcl&quot;
    (&quot;package ifneeded Itcl 3.4&quot; script)
    invoked from within
&quot;package require Itcl 3.4&quot;
    invoked from within
&quot;set ::PKG(Itcl) [package require Itcl 3.4]&quot;
    (file &quot;/sqlitestudio-2.1.0.bin/lib/sqlitestudio/main.tcl&quot; line 209)
    invoked from within
&quot;source /sqlitestudio-2.1.0.bin/lib/sqlitestudio/main.tcl&quot;
    (&quot;package ifneeded sqlitestudio 2.1.0&quot; script)
    invoked from within
&quot;package require sqlitestudio&quot;
    (file &quot;/sqlitestudio-2.1.0.bin/main.tcl&quot; line 6)
\n\n**Operating system:**\nLinux 32-bit\n\n**Version:**\n2.1.0

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

Try this:
1. Download http://sqlitestudio.one.pl/temp/tclkit
2. Make it executable (chmod +x tclkit)
3. Run it: ./tclkit
4. It will become an interactive Tcl shell. Type: package require Itcl

Tell me if this causes the same error for you or not?

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

Thank you for your response.
Here are the results:

# ./tclkit
% package require Itcl
3.4
%

So it looks like it works in this case.

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

Are your both Linux systems 32-bit?

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

_Status changed to **assigned**_

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

Yes, 32 bit. Also, my kernel version is 3.2.6, and the tcl version installed is 8.5.9 (but that shouldn't matter). I'm happy to provide any other information I can to help you find the issue, just let me know how I can assist. 

Thanks

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

I'm not sure if it would help or if it would be overkill, but I could make a virtualbox image (or a livecd iso) available that reproduces the issue.

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

Yes, that would help a lot!

Do you have any method to share it once it's ready?

**Comment by pawelsalawa** [comment]
Originally created at: Sat Jan  5 00:50:27 2013

You know, I think I figured it out!
It was while I was preparing the image.
The issue only occurs when I am within a chroot. If I leave the chroot and try again, everything works fine. Surprisingly, this was the first time I tried it on my test system outside of a chroot (I probably should have done that first...)

Tt doesn't look like this is a real issue after all, so it can probably be closed.

Thank you for your time!
